https://github.com/mozilla/rr/pull/1981

From fefdb3b89e98192601dfdc98c8927c9a91f07f9f Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Mon, 13 Feb 2017 16:01:52 -0500
Subject: [PATCH] tests: include sys/sysmacros.h for minor/major funcs

Newer versions of glibc are deprecating the implicit sys/sysmacros.h
include via sys/types.h, so include it explicitly.
---
 src/test/rrutil.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/test/rrutil.h b/src/test/rrutil.h
index 60de6e47a57b..bb5b527c4c9a 100644
--- a/src/test/rrutil.h
+++ b/src/test/rrutil.h
@@ -76,6 +76,7 @@
 #include <sys/socket.h>
 #include <sys/stat.h>
 #include <sys/sysinfo.h>
+#include <sys/sysmacros.h>
 #include <sys/time.h>
 #include <sys/timerfd.h>
 #include <sys/times.h>
-- 
2.11.0

